NuRadioMC/SignalProp/analyticraytracing.py

```diff
@@ -1744,7 +1749,362 @@ def get_path(self, iS, n_points=1000):
         dP = path_2d - np.array([self.__X1[0], 0, self.__X1[2]])
         MM = np.matmul(self.__R.T, dP.T)
         path = MM.T + self.__X1
-        return path
+        return path        
+                    
+    def get_effective_index(self, n, direction, nx, ny, nz):
+        """
+        objective function to find the possible values of 'n' into the direction of 'k'
```

Comment:
Could you add a bit more doc string. I am not sure that I follow what "possible values" mean.

Reply:
Also please describe the output (format and meaning)